### ISSUE TITLE
In a Listener the attribute "doc" is always empty for start_test

### ISSUE THREAD

**Issue by kootstra** [robotframework/robotframework#2506]
When creating a listener the event method receives a set of attributes. For the start_suite the `doc` attribute is filled with the Documentation of the suite. 

For the start_test the doc attibute is there, but it's always empty. I would have expected this to be mapped to the Documentation of the individual test case. Below is the received attributes list.
```
'{critical': 'yes',
 'doc': u'Test case level documentation',
 'id': 's1-s1-t1',
 'longname': u'TestB.Test Pass Fail Etc Matrix.Force Pass trivial',
 'starttime': '20161229 11:12:53.311',
 'tags': [u'DIM-002',
          u'feature:my_first_feature',
          u'story:my_first_story',
          u'trivial'],
 'template': ''}
```

**Comment by pekkaklarck** [comment]
At least the included example clearly has `doc` with value `Test case level documentation`. Or did I get something wrong? Could you clarify?

**Comment by kootstra** [comment]
You didn't get anything wrong and I'm virtually face slapping myself for not seeing this earlier when I posted it. Let's put it down as reduced focus due to a long session of intense working. 

In any case, it now appears I've uncovered a bug in Allure, as there are two sections showing the test-case/test-steps overview and the description for the test-case tag is shown in one, but not the other. Seems that they're not reusing code as much as they should.
